tianlutech/mediterranea-marine-nextjs
# DBT-1047 Feat/error page

## Context


Here we are adding an error page for user to see when there is an error



[Task](https://www.notion.so/tianlu/Error-Page-Booking-already-signed-Google-Maps-Fails-4184b5b7edb84ce3b943993f04fda96f?pvs=4)

## Test

- Try removing the google API key and see the error message
- Try Booking with a booking which already exists
- Try with a page which doesn't exists





## Platform
Google chrome


## Proof of Concept
<img width="612" alt="image" src="https://github.com/tianlutech/mediterranea-marine-nextjs/assets/79788219/6020b19f-161a-4a1b-8d7c-5a7dc61c615a">
